wpilibc/sim/src/IterativeRobot.cpp

```diff
@@ -152,20 +111,8 @@ void IterativeRobot::StartCompetition() {
 
 /**
  * Determine if the periodic functions should be called.
- *
- * If m_period > 0.0, call the periodic function every m_period as compared
- * to Timer.Get().  If m_period == 0.0, call the periodic functions whenever
- * a packet is received from the Driver Station, or about every 20ms.
- *
- * @todo Decide what this should do if it slips more than one cycle.
  */
-
 bool IterativeRobot::NextPeriodReady() {
```

Comment:
Seems like we should remove NextPeriodReady() entirely as well, since it's private and no longer serves a purpose.